lib/galaxy/webapps/galaxy/api/__init__.py

```diff
@@ -198,7 +198,9 @@ def base(self) -> str:
 
     @property
     def host(self) -> str:
-        return str(self.__request.client.host)
+        host = self.__request.client.host
+        assert host is not None
```

Comment:
Shouldn't this be
```suggestion
        client = self.__request.client
        assert client is not None
        return str(client.host)
```
?